Spartan/spartan.munki.recipe

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads the latest version of Spartan from www.wavefun.com and imports it into Munki</string>
+	<key>Identifier</key>
+	<string>com.github.its-unibas.munki.spartan</string>
+	<key>Input</key>
+	<dict>
+		<key>NAME</key>
+		<string>Spartan</string>
+		<key>SEARCH_PATTERN3</key>
+		<string>"http:\/\/downloads\.wavefun\.com\/Mac\/Spartan18V145\.dmg".*Macintosh.*v(\..\..\..)</string>
+		<key>SEARCH_URL</key>
+		<string>https://www.wavefun.com/downloads</string>
+		<key>MUNKI_REPO_SUBDIR</key>
+		<string>apps/spartan</string>
+		<key>pkginfo</key>
+		<dict>
+			<key>catalogs</key>
+			<array>
+				<string>testing</string>
+			</array>
+			<key>category</key>
+			<string>Productivity</string>
+			<key>description</key>
+			<string>Desktop application for chemistry research and academia.</string>
+			<key>developer</key>
+            <string>Wavefunction Inc.</string>
+            <key>display_name</key>
+            <string>%NAME%</string>
+            <key>minimum_os_version</key>
+            <string>10.12.0</string>
+            <key>name</key>
+            <string>%NAME%</string>
+            <key>unattended_install</key>
+            <true />
+		</dict>
+	</dict>
+	<key>MiniumumVersion</key>
+	<string>1.0</string>
+	<key>ParentRecipe</key>
+	<string>com.github.its-unibas.download.spartan</string>
+	<key>Process</key>
+	<array>
+		<dict>
+			<key>Arguments</key>
+				<dict>
+					<key>re_pattern</key>
+					<string>%SEARCH_PATTERN3%</string>
+					<key>url</key>
+					<string>%SEARCH_URL%</string>
+					<key>result_output_var_name</key>
+					<string>MINOR</string>
```

Comment:
I don't think this is needed. The downloaded dmg contains a app-bundle and we should be able to get the version from there.